src/components/Feeds/BusinessInNeedFeed.js

```diff
@@ -3,28 +3,45 @@ import { StaticQuery, graphql } from 'gatsby';
 
 import { Box } from '@chakra-ui/core';
 
-const SupportingOrgs = data => {
-  return <Box as="pre">{JSON.stringify(data, null, 2)}</Box>;
+const Businesses = data => {
+  return (
+    <Box as="ul">
+      {data.allAirtableBizInNeed.edges.map(({ node: { data } }) => (
```

Comment:
AH! I think this is failing on the frontend because the data returned from Airtable comes down with a `data` object.

`{data.data.allAirtableBizInNeed.edges.map....` seems to fix the issue @kkemple

Reply:
I'll update first thing in the morning! It was working locally so also confused!

Reply:
@kkemple I may be able to get the work you completed here wrapped in with the actual Card component scaffold here: https://github.com/Rebuild-Black-Business/RBB-Website/pull/33 if that's alright with you!

Reply:
Yeah totally! Didn't realize there was one!!

Reply:
@kkemple these folks are moving at an insane pace, so no worries at all. Just pinged that other ticket with your work here. Thank you again!

Reply:
Technically speaking. It's a `data` prop that's being passed into the component. I think the real culprit here is this:

```jsx
// what was written:
const Businesses = data => {
  data.data.airtableStuff
}

// what that actually is:
const Businesses = props => {
  props.data.airtableStuff
}
```